packages/terra-clinical-header/src/Header.module.scss

```diff
@@ -1,77 +1,79 @@
+// import themes
+@import './clinical-lowlight-theme/Header.module';
+@import './orion-fusion-theme/Header.module';
+
 :local {
   .flex-header {
-    background-color: var(--terra-clinical-header-background-color, #f4f4f4);
-    border-bottom-color: var(--terra-clinical-header-border-bottom-color, #dedfe0);
-    border-bottom-style: solid;
-    border-bottom-width: 1px;
+    align-items: var(--terra-clinical-header-align-items, flex-start);
+    background-color: var(--terra-clinical-header-background-color, #f9f9f9);
+    border-bottom: var(--terra-clinical-header-border-bottom, 1px solid #d3d4d5);
+    color: var(--terra-clinical-header-color);
     display: flex;
-    height: var(--terra-clinical-header-height, 3.587rem);
     justify-content: space-between;
-    overflow: hidden;
-    padding: var(--terra-clinical-header-padding, 0.714rem);
+    min-height: var(--terra-clinical-header-min-height, 2.92857rem);
+    padding: var(--terra-clinical-header-padding, 0.35714rem 0.35714rem 0.35714rem 0);
     position: relative;
     width: 100%;
 
     .flex-collapse {
-      align-items: center;
+      align-self: var(--terra-clinical-header-flex-collapse-align-self);
```

Comment:
Does this need a default fallback or is it only used within other themes?

Reply:
This is only used by the fusion-theme

Reply:
update backed on feedback from Jeremy: https://github.com/cerner/terra-clinical/pull/641/commits/daeb48597a485300015085b1bccc2a5725d42829